### ISSUE TITLE
Fix hardcoded conjunction in precisedelta()

### ISSUE THREAD

**Issue by webkaiyo** [jmoiron/humanize#161]
There's a hardcoded conjunction in [line 498 - time.py](https://github.com/jmoiron/humanize/blob/eb9a9453ea35dce5529ed0306865f5842f5bcf3d/src/humanize/time.py#L498) which needs to be fixed.

**Comment by hugovk** [comment]
Thanks, please could you provide some example code which triggers it?

**Comment by webkaiyo** [comment]
Hi, @hugovk. Every code will trigger because it is hardcoded conjunction. Example:

```py
>>> humanize.i18n.activate('pt_BR')
<gettext.GNUTranslations object at 0x03652AD8>
>>> humanize.precisedelta(timedelta(milliseconds=127_000))
'2 minutos and 7 segundos' # it should be '2 minutos e 7 segundos'.
>>> humanize.i18n.activate('ru_RU')
<gettext.GNUTranslations object at 0x03652C40>
>>> humanize.precisedelta(timedelta(milliseconds=127_000))
'2 минуты and 7 секунд' # it should be '2 минуты и 7 секунд'.
```
As you can see, there's a hardcoded `and` conjunction which needs to be fixed.

**Comment by hugovk** [comment]
Thanks! Please see PR #163 for a fix, cc @eldipa.

**Comment by webkaiyo** [comment]
Thank you! I'll test this patch update.

**Comment by webkaiyo** [comment]
Hmm, I don't know why, but the problem still persists.

I used this command line to install the patch branch:
```
pip install git+https://github.com/hugovk/humanize@i18n-and
```

And this command line to check if the problem was solved:
```py
>>> import humanize
>>> humanize.__version__
'1.0.1.dev167'
>>> humanize.activate('pt_BR')
<gettext.GNUTranslations object at 0x03892AD8>
>>> from datetime import timedelta
>>> humanize.precisedelta(timedelta(seconds=122))
'2 minutos and 2 segundos'
```


Did I do something wrong?

**Comment by hugovk** [comment]
Thanks for testing! That's because I forgot to add the pt_BR translation here:

https://github.com/jmoiron/humanize/pull/163/commits/af178e484874c6bd19082445510e38d075038123#diff-b45af0ec59e02401849a9249157c36b0R286

I've now added it to both pt_BR and pt_PT: https://github.com/jmoiron/humanize/pull/163/commits/6bf67c735b33a2b054456ab00de93934682bbc2f.

If you happen to know any of these languages, we could update those at the same time:

```
src/humanize/locale/es_ES/LC_MESSAGES/humanize.po
src/humanize/locale/fa_IR/LC_MESSAGES/humanize.po
src/humanize/locale/id_ID/LC_MESSAGES/humanize.po
src/humanize/locale/it_IT/LC_MESSAGES/humanize.po
src/humanize/locale/ja_JP/LC_MESSAGES/humanize.po
src/humanize/locale/ko_KR/LC_MESSAGES/humanize.po
src/humanize/locale/pl_PL/LC_MESSAGES/humanize.po
src/humanize/locale/sk_SK/LC_MESSAGES/humanize.po
src/humanize/locale/tr_TR/LC_MESSAGES/humanize.po
src/humanize/locale/uk_UA/LC_MESSAGES/humanize.po
src/humanize/locale/vi_VI/LC_MESSAGES/humanize.po
src/humanize/locale/zh_CN/LC_MESSAGES/humanize.po
```

**Comment by webkaiyo** [comment]
Tested right now and it seems to work perfectly now. Thank you for the update! Let me know when this branch is merged so I can update the library.

**Comment by hugovk** [comment]
Released in 3.0.0!

https://github.com/jmoiron/humanize/releases/tag/3.0.0
